purview/api/v1.py

```diff
@@ -17,3 +20,15 @@ def about():
     :return: A JSON object with version info
     """
     return jsonify({'version': version})
+
+
+@api_v1.route('/<resource>/<uid>')
+def get_resource(resource, uid):
+    for model in all_models:
+        if model.__name__.lower() == resource:
+            for prop_name, prop_def in model.defined_properties().items():
+                if isinstance(prop_def, UniqueIdProperty):
+                    item = model.nodes.get(**{prop_def.name: uid})
```

Comment:
You should either change `model.nodes.get` to `model.nodes.get_or_none` or add an exception handler to Flask for `neomodel.DoesNotExist`.